kivy/uix/slider.py

```diff
@@ -206,6 +213,17 @@ def on_touch_down(self, touch):
                     self.value = max(
                         self.min,
                         self.value - (self.max - self.min) / 20)
+        if self.sensitive == 'cursor':
+            for child in self.canvas.children:
+                if isinstance(child, Rectangle):
```

Comment:
Thanks for taking the time to have a look at my PR!
I absolutely agree with you and I knew that this hack may raise some eyebrows but I didn't find any simple way of inferring the cursor area.
Is there already a reference implementation of widgets exposing their sensitive area(s) or should I get back to work and cook my own?